src/main/kotlin/io/github/inductiveautomation/kindling/log/WrapperLogView.kt

```diff
@@ -47,15 +47,19 @@ class WrapperLogView(
 }
 
 object LogViewer : MultiTool, ClipboardTool {
+    private const val MAX_EXTENSION_INDEX = 20
```

Comment:
Has this actually come up before? It's not like counts > 5 can't be opened, they just have to switch to 'all files', right?

Reply:
Surprisingly, yes. It's pretty common to get up to 10. I asked around and someone got 100 at one point (which is almost absurd). The all files method does work, but this makes it easier for drag and drop.